### DISCUSSION TITLE
GoDaddy ..

### DISCUSSION THREAD

**Discussion by jgross-OBA** [polymind-inc/acmebot#307] [Q&A] [answered]
It apprears that the documentation now has the GoDaddy DNS provider as does the source code but it seems that the version in the deployment, 3.5.6, does not yet have GoDaddy integration. 

Can you let us know when this has been integrated as we are looking to utilize this.

Thank you

**Comment by shibayan** [comment, marked as answer]
A GoDaddy-enabled version will be released soon; README was published earlier.
